tests/Integration/Subscriptions/StorageManagerTest.php

```diff
@@ -20,13 +27,115 @@ class StorageManagerTest extends TestCase implements GraphQLContext
      */
     protected $storage;
 
+    protected function getPackageProviders($app)
+    {
+        return array_merge(
+            parent::getPackageProviders($app),
+            [SubscriptionServiceProvider::class]
+        );
+    }
+
     protected function setUp(): void
     {
         parent::setUp();
 
         $this->storage = app(StorageManager::class);
     }
 
+    protected function getEnvironmentSetUp($app)
+    {
+        parent::getEnvironmentSetUp($app);
+
+        $app->bind(ContextSerializer::class, function (): ContextSerializer {
+            return new class() implements ContextSerializer {
+                /**
+                 * Serialize the context.
+                 *
+                 * @param  \Nuwave\Lighthouse\Support\Contracts\GraphQLContext $context
+                 * @return string
+                 */
+                public function serialize(GraphQLContext $context)
+                {
+                    return 'foo';
+                }
+
+                /**
+                 * Unserialize the context.
+                 *
+                 * @param  string $context
+                 * @return \Nuwave\Lighthouse\Support\Contracts\GraphQLContext
+                 */
+                public function unserialize(string $context)
+                {
+                    return new class() implements GraphQLContext {
+                        /**
+                         * Get an instance of the authenticated user.
+                         *
+                         * @return \Illuminate\Foundation\Auth\User|null
+                         */
+                        public function user()
+                        {
+                            //
+                        }
+
+                        /**
+                         * Get an instance of the current HTTP request.
+                         *
+                         * @return \Illuminate\Http\Request
+                         */
+                        public function request()
+                        {
+                            //
+                        }
+                    };
+                }
+            };
+        });
+    }
+
+    /**
+     * Construct a dummy subscriber for testing.
+     *
+     * @param  string  $queryString
+     * @return \Nuwave\Lighthouse\Subscriptions\Subscriber
+     */
+    protected function subscriber(string $queryString): Subscriber
+    {
+        /** @var Subscriber $subscriber */
+        $subscriber = $this->getMockBuilder(Subscriber::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $subscriber->channel = Subscriber::uniqueChannelName();
+        $subscriber->query = Parser::parse($queryString);
+
+        return $subscriber;
+    }
+
+    /**
+     * @see \Nuwave\Lighthouse\Support\Contracts\GraphQLContext::user()
+     *
+     * @return \Tests\Utils\Models\User
+     */
+    public function user(): User
+    {
+        return new User([
```

Comment:
Would use factory make

Reply:
Can you elaborate on that?

Reply:
```php
return factory(User::class)->make([
    'first_name' => 'John',
    'last_name' => 'Doe',
    'email' => 'john@doe.com',
]);
```

Reply:
Just realized we can remove the method entirely